libraries/AC_WPNav/AC_WPNav.cpp

```diff
@@ -10,7 +10,7 @@ const AP_Param::GroupInfo AC_WPNav::var_info[] = {
     // @DisplayName: Waypoint Horizontal Speed Target
     // @Description: Defines the speed in cm/s which the aircraft will attempt to maintain horizontally during a WP mission
     // @Units: cm/s
-    // @Range: 0 2000
+    // @Range: 20 2000
```

Comment:
this is wrong now, as it is user settable, so you can set 0 to speed if you want !